packages/contentstack-clone/src/lib/util/abort-controller.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const { EventEmitter } = require('events');
+
+class CustomAbortSignal {
+  constructor() {
+    this.eventEmitter = new EventEmitter();
+    this.onabort = null;
+    this.aborted = false;
+  }
+  toString() {
+    return '[object CustomAbortSignal]';
+  }
+  get [Symbol.toStringTag]() {
+    return 'CustomAbortSignal';
+  }
+  removeEventListener(name, handler) {
+    this.eventEmitter.removeListener(name, handler);
+  }
+  addEventListener(name, handler) {
+    this.eventEmitter.on(name, handler);
+  }
+  dispatchEvent(type) {
+    const event = { type, target: this };
+    const handlerName = `on${type}`;
+
+    if (typeof this[handlerName] === 'function') this[handlerName](event);
+
+    // this.eventEmitter.emit(type, event);
```

Comment:
@vkalta Avoid pushing unused/commented code

Reply:
@netrajpatel This is resolved.